react/features/settings/actions.web.ts

```diff
@@ -200,6 +201,20 @@ export function submitProfileTab(newState: any) {
     };
 }
 
+/**
+* Changes the display name for the local user.
+*
+* @param {string} nickname - The new display name.
+* @returns {Function}
+*/
+export function changeLocalDisplayName(nickname = '') {
```

Comment:
Can you please check how we handle this on native, which we already do?

Reply:
The whole display name update seems a little bit messy. Both mobile and web use `updateSettings` to update it. It is called in quite a lot of places in mobile and web. Actually this action also dispatches `updateSettings` but just uses `getNormalizedDisplayName` for the display name value before this. 

In the code in some places we don't transform the display name value at all, in other - we trim it, we escape it, and here we use `getNormalizedDisplayName` which will normalize it and actually put a limit on the number of chars. 

It looks to me that maybe we would want to use `getNormalizedDisplayName` for all the places where we change the local display name. Maybe we can add escape on some or all of the places. Anyway it seems this is a totally different refactoring and task.

My proposal is to leave this action there, so that when we check what kind of formatting we want for all the places where we update the local display name, we can put it in this action and start using only this action for local display name update. 

WDYT?

Reply:
Honestly I'd rather not add an action we already know will go away. You could inline it, as in, call `updateSettings` with `getNormalizedDisplayName`, and we can take it as a separate task to review that behavior everywhere else.

Reply:
What I actually meant is that I expect the action to be used in all places. But anyway I don't mind removing it for now too. So I'll remove it!